modules/understanding-upgrade-channels.adoc

```diff
@@ -7,104 +7,97 @@
 
 [id="understanding-upgrade-channels_{context}"]
 
-= Upgrade channels and release paths
-Cluster administrators can configure the upgrade channel from the web console.
-
+= Update channels
 
 [id="candidate-version-channel_{context}"]
-== candidate-{product-version} channel
-
-The `candidate-{product-version}` channel contains candidate builds for a z-stream ({product-version}.z) and previous minor version releases. Release candidates contain all the features of the product but are not supported. Use release candidate versions to test feature acceptance and assist in qualifying the next version of {product-title}. A release candidate is any build that is available in the candidate channel, including ones that do not contain link:https://semver.org/spec/v2.0.0.html#spec-item-9[a pre-release version] such as `-rc` in their names. After a version is available in the candidate channel, it goes through more quality checks. If it meets the quality standard, it is promoted to the `fast-{product-version}` or `stable-{product-version}` channels. Because of this strategy, if a specific release is available in both the `candidate-{product-version}` channel and in the `fast-{product-version}` or `stable-{product-version}` channels, it is a Red Hat-supported version. The `candidate-{product-version}` channel can include release versions from which there are no recommended updates in any channel.
```

Comment:
Original phrasing too wordy (and puts the important bits at the end, i.e. not supported):
Release candidates contain all the features of the product but are not supported. 

Suggestion:
Release candidates are not supported, but they contain all product features.

Reply:
we are also trying to move away from "but they contain all product features" for prereleases, as we approach 4.13 and possibly putting 4.13.0-ec.# into a `candidate-4.13` channel.  Those engineering candidates will still be in the prerelease channel, but because we'll be cutting them basically as soon as 4.12 forks off, they won't contain all the features that will eventually be part of 4.13.

Reply:
I'll try to clean it up a bit below and tag you for comment there.